packages/common/container-definitions/src/loader.ts

```diff
@@ -533,10 +533,8 @@ export interface IHostLoader extends ILoader {
 /**
  * @public
  */
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
 export type ILoaderOptions = {
```

Comment:
i like removing it, but i think its used today, so we need a transition path, i don't think we can just unilaterally remove